NOTICE.txt

```diff
@@ -2636,6 +2663,19 @@ SOFTWARE.
 
 ---
 
+## semver
+
+This product contains 'semver' by semver.
+
+Semantic Versioning Specification
+
+* HOMEPAGE:
+  * https://github.com/semver/semver
+
+* LICENSE: 
```

Comment:
Doesn't have a license.

Reply:
Where are you getting this from? I don't see it.

Reply:
I got it from here: https://github.com/mattermost/mattermost-webapp/commit/e709f99ccde988bd9566f56b7fa6f8a6253444f5#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

But I don't see it in package.json anymore either so maybe it was removed.

Reply:
@crspeller It is being re-added to v5.16 with this PR: https://github.com/mattermost/mattermost-webapp/pull/3807.

Reply:
Alright. But if it doesn't have a license I don't think we can use it. The default is all rights reserved.

Reply:
Hey @amyblais, the package URL is actually `https://github.com/npm/node-semver` and it's license [is here](https://github.com/npm/node-semver/blob/master/LICENSE).

Reply:
Thank you :)